File: Java/Petshop.java
public class Petshop {
    private int id;
    private String namaProduk;
    private int hargaProduk;
    private int stokProduk;

    public Petshop() 
    {
    }

    // Constructor
    public Petshop(int id, String nama, int hargaProduk, int stokProduk) {
        this.id = id;
        this.namaProduk = nama;
        this.hargaProduk = hargaProduk;
        this.stokProduk = stokProduk;
    }

    // Getter
    public int getId() 
    {
        return id;
    }

    public String getNama() 
    {
        return namaProduk;
    }

    public int getHargaProduk() 
    {
        return hargaProduk;
    }   

    public int getStokProduk() 
    {
        return stokProduk;
    }

    // Setter
    public void setId(int id) 
    {
        this.id = id;
    }

    public void setNama(String nama) 
    {
        this.namaProduk = nama;
    }

    public void setHargaProduk(int hargaProduk) 
    {
        this.hargaProduk = hargaProduk;
    }

    public void setStokProduk(int stokProduk) 
    {
        this.stokProduk = stokProduk;
    }

}

File: PHP/Petshop.php
<?php

class Petshop
{
    private $id;
    private $nama_produk;
    private $stok_produk;
    private $harga_produk;
    private $foto_produk;

    public function __construct($id, $nama_produk, $stok_produk, $harga_produk, $foto_produk)
    {
        $this->id = $id;
        $this->nama_produk = $nama_produk;
        $this->stok_produk = $stok_produk;
        $this->harga_produk = $harga_produk;
        $this->foto_produk = $foto_produk;
    }

    public function setId($id)
    {
        $this->id = $id;
    }

    public function getId()
    {
        return $this->id;
    }

    public function setNamaProduk($nama_produk)
    {
        $this->nama_produk = $nama_produk;
    }

    public function getNamaProduk()
    {
        return $this->nama_produk;
    }

    public function setStokProduk($stok_produk)
    {
        $this->stok_produk = $stok_produk;
    }

    public function getStokProduk()
    {
        return $this->stok_produk;
    }

    public function setHargaProduk($harga_produk)
    {
        $this->harga_produk = $harga_produk;
    }

    public function getHargaProduk()
    {
        return $this->harga_produk;
    }

    public function setFotoProduk($foto_produk)
    {
        $this->foto_produk = $foto_produk;
    }

    public function getFotoProduk()
    {
        return $this->foto_produk;
    }

    // Destructor
    public function __destruct()
    {
    }
}